src/Components/Drink.js

```diff
@@ -56,11 +56,11 @@ const Drink = () => {
     })
 
     return (
-        <div className="paired-component">
+        <div aria-label="Random Drink Result" className="paired-component">
             <h3 className='drink-name'>{currentDrink.strDrink}</h3>
-            <section className='drink-info'>
-                <img className="drink-image" src={currentDrink.strDrinkThumb} />
-                <aside className='drink-specs'>
+            <section aria-label="Drink Information" className='drink-info'>
+                <img alt={`${currentDrink.strDrink} image`} className="drink-image" src={currentDrink.strDrinkThumb} />
```

Comment:
Nice use of the state here @colganmeanor !

Reply:
Agreed - smart thinking!